libsolidity/formal/Predicate.cpp

```diff
@@ -316,7 +327,8 @@ optional<string> Predicate::expressionToString(smtutil::Expression const& _expr,
 {
 	if (smt::isNumber(*_type))
 	{
-		solAssert(_expr.sort->kind == Kind::Int, "");
+		if (_expr.sort->kind != Kind::Int)
+			solAssert(false, "");
```

Comment:
Hmm, how is this version better than the orig?

Reply:
Ah, it's a bug ;) changed for debugging, will revert